client/my-sites/plan-features-2023-grid/billing-timeframe.jsx

```diff
@@ -22,11 +22,10 @@ function getPerMonthDescription( {
 	}
 
 	if ( ! isMonthlyPlan ) {
-		const annualPriceObj = getCurrencyObject( rawPriceAnnual, currencyCode );
-		const annualPriceText = `${ annualPriceObj?.symbol }${ annualPriceObj?.integer }`;
+		const annualPrice = formatCurrency( rawPriceAnnual, currencyCode );
 
-		return translate( 'per month, %(annualPriceText)s billed annually', {
-			args: { annualPriceText },
+		return translate( 'per month, %(annualPrice)s billed annually', {
```

Comment:
We can maintain the same translated string here and just change how the variable is generated (otherwise it will require a new translation).